### DISCUSSION TITLE
How can I control the parameters exported for visualization ?

### DISCUSSION THREAD

**Discussion by gauravity30** [xcompact3d/Incompact3d#213] [Q&A] [answered]
Hello,
I wanted to know how can I control the paramters that are exported for visualization ? By default, the `ux, uy, uz, pp, critq, vort` are exported. I don't want the `critq` parameter to be exported. How can acheive this ?

Thanks,
Gaurav

**Comment by mathrack** [comment, marked as answer]
Hello,

The default setup is to output velocity, scalars (density) and the pressure. Some cases will also output the vorticity and / or the Q criterion. For instance, the cylinder case will output vorticity and the Q criterion. The channel flow case will only output the Q criterion. Compare both cases and you will find your answer.
